backend/controllers/indUsersController.js

```diff
@@ -0,0 +1,25 @@
+const IndUser = require('../models/indUserModel');
+const mongoose = require('mongoose');
+
+//get all individual users
+const getAllIndUsers = async (req, res) => {
```

Comment:
There wouldn't be a case where we would need to getAllIndUsers, but we would need a getIndUserByID

Reply:
Aah, I left a comment about this on Ryan's code - I thought we were keeping it so we could easily test on Postman?

Reply:
yeah i feel like its good to leave in, we can take out if/before we go live